### ISSUE TITLE
Conflicting function names

### ISSUE THREAD

**Issue by mattssca** [morinlab/GAMBLR.utils#6]
When loading GAMBLR.utils I get a warning message about conflicting function names. This can be resolved by specifying the Import field in the function documentation. 

The conflicting packages are; `IRanges` and `stats` and the functions affected are: `start` and `end`. 

Without looking into this much further, I assume that any function that imports `IRanges` (e.g [annotate_ssm_motif_ccontext](https://github.com/morinlab/GAMBLR.utils/blob/main/R/annotate_ssm_motif_context.R)) should have the  `@rawNamespace import` updated to also exclude `start`, and `end`.  Given that this function is not using these functions from IRanges.

```
@rawNamespace import(IRanges, except = c("start", "end", "merge", "shift", "collapse", "union", "slice", "intersect", "setdiff", "desc", "reduce")) 
```

The warning message:
```
> library(GAMBLR.utils)
Warning messages:
1: replacing previous import ‘IRanges::end’ by ‘stats::end’ when loading ‘GAMBLR.utils’ 
2: replacing previous import ‘IRanges::start’ by ‘stats::start’ when loading ‘GAMBLR.utils’ 
```

It's crucial to suppress warnings like this since it will cause the build-check by Git Actions to fail.

**Comment by mattssca** [comment]
This logic also applies to the import of `GenomicRanges` in the [same function](https://github.com/morinlab/GAMBLR.utils/blob/main/R/annotate_ssm_motif_context.R#L20);
```
@rawNamespace import(GenomicRanges, except = c("start", "end", "merge", "shift", "union", "intersect", "setdiff", "reduce"))
```

**Comment by vladimirsouza** [comment]
The issue was addressed in PR [#7](https://github.com/morinlab/GAMBLR.utils/pull/7).
